Library/Formula/jhead.rb

```diff
@@ -1,14 +1,16 @@
-require 'formula'
-
 class Jhead < Formula
-  homepage 'http://www.sentex.net/~mwandel/jhead/'
-  url 'http://www.sentex.net/~mwandel/jhead/jhead-2.97.tar.gz'
-  sha1 'ca4965a19d60078a3fe2cfb6d3635a083f958f2e'
+  homepage "http://www.sentex.net/~mwandel/jhead/"
+  url "http://www.sentex.net/~mwandel/jhead/jhead-3.00.tar.gz"
+  sha1 "6bd3faa38cc884b5370e8e8f15bc10cbb706ec7a"
 
   def install
     system "make"
     bin.install "jhead"
```

Comment:
Would be good to submit a patch to upstream to fix `make install` so we can use it.